static/app/data/platformCategories.tsx

```diff
@@ -517,17 +517,17 @@ const customMetricFrontendPlatforms: readonly PlatformKey[] = [
 ];
 
 // These are all the platforms that can set up custom metrics.
-export const customMetricPlatforms: Set<PlatformKey> = new Set([
+export const customMetricPlatforms: readonly PlatformKey[] = [
   ...customMetricFrontendPlatforms,
   ...customMetricBackendPlatforms,
-]);
+];
 
 /**
  * The list of platforms for which we have created onboarding instructions.
  * Should be a subset of the list of `customMetricPlatforms`.
  */
-export const customMetricOnboardingPlatforms = new Set(
-  [...customMetricPlatforms].filter(
+export const customMetricOnboardingPlatforms: readonly PlatformKey[] =
+  customMetricPlatforms.filter(
```

Comment:
>converting these from sets to readonly arrays to match the other platform arrays being passed into the hook

nice, it also helps here to have an Array instead of a Set